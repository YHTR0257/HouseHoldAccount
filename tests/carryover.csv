Date,ID,SubjectCode,Amount,Remarks,Subject,Year,Month,YearMonth
2024-03-03,20240303010,200,-2000,Starbucks 01,Credit Card debt,2024,3,2024-03
2024-03-03,20240303011,500,2000,Starbucks 01,Groceries,2024,3,2024-03
2024-03-03,20240303020,101,-1500,Tempura 02,UFJ,2024,3,2024-03
2024-03-03,20240303021,500,1500,Tempura 02,Groceries,2024,3,2024-03
2024-03-10,20240310010,101,-10000,UFJ 01,UFJ,2024,3,2024-03
2024-03-10,20240310011,590,10000,Special Expenses 01,Special Expenses, 2024,3,2024-03
2024-03-15,20240315010,400,-5000,Salary 01,Salary,2024,3,2024-03
2024-03-15,20240315011,101,5000,Salary 01,UFJ,2024,3,2024-03
2024-03-20,20240320010,490,-500,Campaign 01,Other income,2024,3,2024-03
2024-03-20,20240320011,200,500,Campaign 01,Credit Card debt,2024,3,2024-03
2024-03-25,20240325010,101,-1000,PayPay 01,UFJ,2024,3,2024-03
2024-03-25,20240325011,500,1000,PayPay 01,Groceries,2024,3,2024-03
