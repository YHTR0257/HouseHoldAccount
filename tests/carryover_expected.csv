Date,ID,SubjectCode,Amount,Remarks,Subject,Year,Month,YearMonth
2024-03-03,20240303010,200,-2000,Starbucks 01,Credit Card debt,2024,3,2024-03
2024-03-03,20240303011,500,2000,Starbucks 01,Groceries,2024,3,2024-03
2024-03-03,20240303020,101,-1500,Tempura 02,UFJ,2024,3,2024-03
2024-03-03,20240303021,500,1500,Tempura 02,Groceries,2024,3,2024-03
2024-03-10,20240310010,101,-10000,UFJ 01,UFJ,2024,3,2024-03
2024-03-10,20240310011,590,10000,Special Expenses 01,Special Expenses, 2024,3,2024-03
2024-03-15,20240315010,400,-5000,Salary 01,Salary,2024,3,2024-03
2024-03-15,20240315011,101,5000,Salary 01,UFJ,2024,3,2024-03
2024-03-20,20240320010,490,-500,Campaign 01,Other income,2024,3,2024-03
2024-03-20,20240320011,200,500,Campaign 01,Credit Card debt,2024,3,2024-03
2024-03-25,20240325010,101,-1000,PayPay 01,UFJ,2024,3,2024-03
2024-03-25,20240325011,500,1000,PayPay 01,Groceries,2024,3,2024-03
2024-03-31,20240331990,300,9000,Carryover 99,Retained Earnings,2024,3,2024-03
2024-03-31,20240331990,600,-9000,Carryover 99,Profit and Loss,2024,3,2024-03
2024-04-01,20240401990,101,-7500,Carryover 99,UFJ,2024,4,2024-04
2024-04-01,20240401990,200,-1500,Carryover 99,Credit Card debt,2024,4,2024-04
2024-04-01,20240401990,300,9000,Carryover 99,Retained Earnings,2024,4,2024-04